ocaml/xapi/message_forwarding.ml

```diff
@@ -922,6 +922,17 @@ functor
         info "Pool.assert_can_enable_tls_verification: pool = '%s'"
           (current_pool_uuid ~__context) ;
         Local.Pool.assert_can_enable_tls_verification ~__context
+
+      let enable_tls_verification ~__context =
+        info "Pool.enable_tls_verification: pool = '%s'"
+          (current_pool_uuid ~__context) ;
+        let self = Helpers.get_pool ~__context in
+        let local_fn = Local.Pool.enable_tls_verification in
```

Comment:
This should call `Local.Pool.assert_can_enable_tls_verification` before trying to enable verification.